src/bref/src/LaravelHttpHandler.php

```diff
@@ -25,42 +24,11 @@ public function __construct(Kernel $kernel)
 
     public function handleRequest(HttpRequestEvent $event, Context $context): HttpResponse
     {
-        Request::setTrustedProxies(['127.0.0.1'], Request::HEADER_X_FORWARDED_FOR | Request::HEADER_X_FORWARDED_HOST | Request::HEADER_X_FORWARDED_PORT | Request::HEADER_X_FORWARDED_PROTO);
-
-        // CGI Version 1.1 - Section 4.1
-        $server = array_filter([
-            'AUTH_TYPE' => $event->getHeaders()['auth-type'] ?? null, // 4.1.1
-            'CONTENT_LENGTH' => $event->getHeaders()['content-length'] ?? null, // 4.1.2
-            'CONTENT_TYPE' => $event->getContentType(), // 4.1.3
-            'QUERY_STRING' => $event->getQueryString(), // 4.1.7
-            'REQUEST_METHOD' => $event->getMethod(), // 4.1.12
-            'SERVER_PORT' => $event->getServerPort(), // 4.1.16
-            'SERVER_PROTOCOL' => $event->getProtocolVersion(), // 4.1.16
-            'DOCUMENT_ROOT' => getcwd(),
-            'REQUEST_TIME' => time(),
-            'REQUEST_TIME_FLOAT' => microtime(true),
-            'REQUEST_URI' => $event->getUri(),
-        ], fn ($value) => null !== $value);
-
-        foreach ($event->getHeaders() as $name => $values) {
-            $server['HTTP_'.strtoupper($name)] = $values[0];
-        }
-
-        // TODO convert request better
-        $request = Request::create(
```

Comment:
Is this an issue that we are now using `Symfony\Component\HttpFoundation\Request` to the Laravel kernel?

Reply:
The interface says that it is accepted.

Reply:
> The interface says that it is accepted.

Don't trust what Laravel phpdoc says. :trollface:

Reply:
You should do:

```php
$request = Illuminate\Http\Request::createFromBase(
    SymfonyRequestBridge::convertRequest($event, $context)
);
```